crates/runtime/benches/setup/mod.rs

```diff
@@ -41,6 +44,11 @@ pub(crate) async fn setup_benchmark(
         () = rt.load_components() => {}
     }
 
+    if acceleration.is_some() {
+        // allow accelerated data to load
+        tokio::time::sleep(std::time::Duration::from_secs(60)).await;
```

Comment:
This will be improved as part of next next PR: add mechanism to detect when all datasets are ready/loaded instead of using hardcoded time.